test/DurableTask.AzureStorage.Tests/AzureStorageScenarioTests.cs

```diff
@@ -2312,6 +2314,40 @@ public async Task ScheduledStart_Activity_GetStatus_Returns_ScheduledStart(bool
             }
         }
 
+        /// <summary>
+        /// Test queue message generated by DTFx.AzureStorage v2.x can be successfully read by DTFx.AzureStorage v1.x
+        /// </summary>
+        [TestMethod]
+        public async Task TestBackwardCompatibilityWithASv2()
+        {
+            //string connectionString = TestHelpers.GetTestStorageAccountConnectionString();
```

Comment:
let's remove commented out code?
```suggestion
            //string connectionString = TestHelpers.GetTestStorageAccountConnectionString();
```